editor/project_manager.cpp

```diff
@@ -483,12 +483,37 @@ class ProjectDialog : public ConfirmationDialog {
 					project_features.sort();
 					initial_settings["application/config/features"] = project_features;
 					initial_settings["application/config/name"] = project_name->get_text().strip_edges();
-					initial_settings["application/config/icon"] = "res://icon.png";
+					initial_settings["application/config/icon"] = "res://icon.svg";
 
 					if (ProjectSettings::get_singleton()->save_custom(dir.plus_file("project.godot"), initial_settings, Vector<String>(), false) != OK) {
 						set_message(TTR("Couldn't create project.godot in project path."), MESSAGE_ERROR);
 					} else {
-						ResourceSaver::save(create_unscaled_default_project_icon(), dir.plus_file("icon.png"));
+						// Store default project icon in SVG format (copied from `editor/icons/DefaultProjectIcon.svg`).
```

Comment:
I had to copy the contents of that file, as we need to save the literal SVG code instead of a rasterized version of the image. Is there any way around this? I don't think the icon will change often, so it's probably not the end of the world if it has to be duplicated.

Reply:
> Is there any way around this?

Instead of deleting, replace `create_unscaled_default_project_icon()` with something like and use it:

```c++

String get_default_project_icon() {
	for (int i = 0; i < editor_icons_count; i++) {
		if (strcmp(editor_icons_names[i], "DefaultProjectIcon") == 0) {
			return String(editor_icons_sources[i]);
		}
	}
	return String();
}

```